DOCUMENTATION.md

```diff
@@ -636,6 +637,7 @@ Click here to expand for more information.
 ```php
 {
     "id" => "template_id",
+    "name" => "Template name"
```

Comment:
Does `name` also need to be added to the responses for 'Get a template by ID and version' and 'Get all templates'?